opm/input/eclipse/Schedule/Group/GSatProd.cpp

```diff
@@ -48,22 +52,37 @@ const GSatProd::GSatProdGroup& GSatProd::get(const std::string& name) const
     return it->second;
 }
 
+const GSatProd::GSatProdGroupProp GSatProd::get(const std::string& name, const SummaryState& st) const
+{
+    using Rate = GSatProdGroupProp::Rate;
+
+    GSatProdGroupProp prop;
+    const GSatProd::GSatProdGroup& group = this->get(name);
+    prop.rate[Rate::Oil] = UDA::eval_group_uda(group.oil_rate, name, st, group.udq_undefined);
+    prop.rate[Rate::Gas] = UDA::eval_group_uda(group.gas_rate, name, st, group.udq_undefined);
+    prop.rate[Rate::Water] = UDA::eval_group_uda(group.water_rate, name, st, group.udq_undefined);
```

Comment:
This does not assign values for `Resv` and `GLift` whence the caller will always get zero for those.  If that's intentional, then please add a comment to explain why.  If not, please add the appropriate values.